app/view_models/paye/PayeYourIncomeAndTaxes.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package view_models.paye
+
+import config.ApplicationConfig
+import models.PayeAtsData
+import view_models.{Amount, Rate}
+
+case class PayeYourIncomeAndTaxes(
+  taxYear: Int,
+  incomeBeforeTax: Amount,
+  taxableIncome: Amount,
+  totalIncomeTax: Amount,
+  incomeAfterTaxNics: Amount,
+  averageTaxRate: String) extends TaxYearFormatting
+
+object PayeYourIncomeAndTaxes {
+
+  val taxYear: Int = ApplicationConfig.payeYear
```

Comment:
Unfortunately, the data doesn't roll over with the tax year, and updates at a later point in the year, so this would require manual intervention to adjust the year of data to display